lightning-background-processor/src/lib.rs

```diff
@@ -238,7 +238,8 @@ impl<
 		G,
 		&'a (dyn UtxoLookup + Send + Sync),
 		L,
-	> where
+	>
```

Comment:
Strangely `cargo fmt` did touch this file even though it wasn't in the exclude list previously.